sample/js/main.js

```diff
@@ -1,17 +1,6 @@
-CKEDITOR.config.extraPlugins = 'structuredheadings';
-CKEDITOR.config.contentsCss = './css/numbering.css';
-CKEDITOR.config.allowedContent = true;
-
-var initSample = ( function() {
-
-	var isStructuredHeadingsAvailable = () => {
-		return !!CKEDITOR.plugins.get('structuredheadings');
-	};
-
-	return function() {
-		var editorElement = CKEDITOR.document.getById( 'editor' );
-		CKEDITOR.replace( 'editor');
-	};
-
-})();
+/* exported initSample */
+CKEDITOR.config.extraPlugins = "structuredheadings";
 
+var initSample = function () {
+  CKEDITOR.replace("editor");

```

Comment:
I know, but I preferred keeping it separate it looks cleaner to me. This file used to do a lot more, honestly I could not only set the config during the call to replace but I could do that in the HTML file and eliminate this js file entirely. I just like to keep things separated. Let me know if you'd like me to change it.